src/prompts/family/neuro.js

```diff
@@ -0,0 +1,28 @@
+import neuro from '../neuro'
+
+export default {
+  ...neuro,
+
+  prompts: [
+    'aw',
+    'hm',
+    'meow',
+    'filtered',
+    'KEKWa',
+    'misgenders someone',
+    'trauma dump',
+    'calls someone an AI',
```

Comment:
```suggestion
    'calls one of them an AI',
```
Prevent orphan text.